<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_product</class>
 <widget class="QWidget" name="Form_product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="image_window">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_exTime">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Mode">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>180</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_CFMode">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>300</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="Exp_spinBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>460</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>35</number>
   </property>
   <property name="maximum">
    <number>350000</number>
   </property>
   <property name="singleStep">
    <number>35</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSlider" name="Exp_horizontalSlider">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>460</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>35</number>
   </property>
   <property name="maximum">
    <number>350000</number>
   </property>
   <property name="singleStep">
    <number>35</number>
   </property>
   <property name="pageStep">
    <number>350</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="explosure_label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>430</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>曝光时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>480</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>相机增益</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Gain_spinBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>500</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>63</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSlider" name="Gain_horizontalSlider">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>500</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>63</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="Mode_comboBox">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>290</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>参数设置</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>设置完成</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_take_photo">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>360</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>拍摄样张</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Start">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>360</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>开始采集</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
